packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -1736,6 +1740,18 @@ export default class NewVm extends BaseComponent {
               </Item>
             </SectionContent>
           ),
+          <SectionContent key='destroyCloudConfigVdi'>
+            <Item>
+              <input
+                checked={destroyCloudConfigVdiAfterBoot}
+                disabled={installMethod === 'noConfigDrive' || !bootAfterCreate}
+                onChange={this._toggleState('destroyCloudConfigVdiAfterBoot')}
+                type='checkbox'
+              />
+              &nbsp;
+              {_('destroyCloudConfigVdiAfterBoot')}
```

Comment:
* use a `<label htmlFor= ... >` to make the label clickable and more accessible
* maybe this checkbox should be grouped with the "boot vm after creation" since it's dependant of this option

Reply:
1. Shouldn't it be `<label for= ... >` instead? :thinking: